hw1_mzw7af_vl4kz.py

```diff
@@ -57,6 +57,30 @@ def main():
     facts = {}
     rules = []
 
+    for line in sys.stdin:
+        argArray = line.strip().split(" ")
 
+        if argArray[0].lower() == "teach":
+            if argArray[3] == "=":
+                stringVar = ' '.join(argArray).split(" = ")[1]
```

Comment:
Doesn't `' '.join(argArray)` give you back the original line? Plz fix. Also don't forget to strip() the line again